epub-doc/fr/src/resources/css/readiumbase.css

```diff
@@ -0,0 +1,406 @@
+}
+
+rt {
+  font-variant-east-asian: ruby;
+}
+
+.frac {
+  font-variant-numeric: diagonal-fractions;
+}
+
+/* We’re asking the rendering engine if it supports CSS custom properties by using a feature query.
+   @supports only checks if the syntax is correct for those CSS custom properties, not if they are valid.
+   We’ll be using the Readium CSS internal framework implementers can use for unstyled EPUBs. */
+
+@supports (--var: a) {
+  :root {
+    --Author-typeScale: 1.125;
+    --Author-baseFontSize: 100%;
+    --Author-baseLineHeight: 1.5rem;
+    --Author-flowSpacing: 1.5rem;
+    --Author-paraSpacing: 1.5rem;
+    --Author-paraIndent: 0;
+    --Author-linkColor: #0000EE;
+    --Author-visitedColor: #551A8B;
+  }
+
+  body {
+    font-size: var(--Author-baseFontSize);
+    line-height: var(--Author-baseLineHeight);
+  }
+
+  /* Flow content */
+
+  blockquote, figure, p, pre,
+  aside, footer, form, hr {
+    margin-top: var(--Author-flowSpacing);
+    margin-bottom: var(--Author-flowSpacing);
+  }
+
+  p {
+    margin-top: var(--Author-paraSpacing);
+    margin-bottom: var(--Author-paraSpacing); 
+    text-indent: var(--Author-paraIndent);
+  }
+
+  :link {
+    color: var(--Author-linkColor);
+  }
+
+  :visited {
+    color: var(--Author-visitedColor);
+  }
+
+  h1 {
+    margin-bottom: calc(var(--Author-flowSpacing) * 2);
+    font-size: calc(((var(--Author-baseFontSize) * var(--Author-typeScale)) * var(--Author-typeScale)) * var(--Author-typeScale));
+  }
+
+  h2 {
+    margin-top: calc(var(--Author-flowSpacing) * 2);
+    margin-bottom: var(--Author-flowSpacing);
+    font-size: calc((var(--Author-baseFontSize) * var(--Author-typeScale)) * var(--Author-typeScale));
+  }
+
+  h3 {
+    margin-top: var(--Author-flowSpacing);
+    margin-bottom: var(--Author-flowSpacing);
+    font-size: calc(var(--Author-baseFontSize) * var(--Author-typeScale));
+  }
+
+  h4 {
+    margin-top: var(--Author-flowSpacing);
+    margin-bottom: var(--Author-flowSpacing);
+    font-size: var(--Author-baseFontSize);
+  }
+
+  h5 {
+    margin-top: var(--Author-flowSpacing);
+    margin-bottom: var(--Author-flowSpacing);
+    font-size: var(--Author-baseFontSize);
+    font-style: normal;
+    font-variant: small-caps;
+  }
+
+  h6 {
+    margin-top: var(--Author-flowSpacing);
+    font-size: var(--Author-baseFontSize);
+    font-weight: bold;
+    font-style: normal;
+    text-transform: lowercase;
+    font-variant: small-caps;
+  }
+
+  /* Lists */
+
+  dl, ol, ul {
+    margin-top: var(--Author-flowSpacing);
+    margin-bottom: var(--Author-flowSpacing);
+  }
+
+  dt {
+    margin-top: var(--Author-flowSpacing);
+  }
+
+  /* Table */
+
+  table {
+    margin: var(--Author-flowSpacing) 0;
+  }
+
+  /* Demonstrates full-width by leveraging Readium CSS variables */
+  .fullbleed {
+	margin: var(--Author-flowSpacing) 0;
+	position: relative;
+	left: 0;
+	left: calc((var(--RS__pageGutter) * var(--USER__pageMargins, 1)) * -1);
```

Comment:
Just a warning: I will remove this from ReadiumCSS docs actually, given ReadiumCSS v2 now applies `overflow: clipped` to `body`. This means the images will be cut off on both sides – this should already be noticeable on playground.